src/game_inventory.cpp

```diff
@@ -667,17 +667,13 @@ class comestible_inventory_preset : public inventory_selector_preset
 
 static std::string get_consume_needs_hint( player &p )
 {
-    const auto &cmgr = get_all_colors();
     auto hint = std::string();
     auto desc = p.get_hunger_description();
-    hint.append( string_format( "[%s <color_%s>%s</color>] ", _( "Food :" ),
-                                cmgr.get_name( desc.second ), desc.first ) );
+    hint.append( string_format( _( "[%s %s] " ), _( "Food :" ), colorize( desc.first, desc.second ) ) );
```

Comment:
Why not this?
```suggestion
    hint.append( string_format( _( "[Food : %s] " ), colorize( desc.first, desc.second ) ) );
```

Reply:
Because `"Food :"` is already a translated string and `"[Food : %s] "` is not, and I did not wanted to create new strings to translate for a simple thing like that.

Also, `"[%s %s] "` is not translated, but will display correctly in all languages,

Reply:
> Also, "[%s %s] " is not translated, but will display correctly in all languages,

Actually, it is translated as it is surrounded by `_(  )`, and translators can change it (but they have no context).

Reply:
I mean, it is translatable, but not *actually* translated, and need not to be for users to understand.

Reply:
I know that making `"[%s %s] "` translatable is not really useful in short term, but I did this to comply with @jbytheway, he had a point when he said:

> Also, it might be worth translating this format string just in case another language wants things formatted differently.

Reply:
It's probably fine in this case, but I've had problems when translating some of the game's text when they are composed with only English semantics in mind... Rather than letting the translators to hack the translation of "Food :" if they want it translated in a different format in "[%s %s]", it's better to just leave the string as the whole so they can do whatever they want with the exact string.

Reply:
Well, 5 strings are not hard translation work... It's definitely doable, whether in this PR or as a separate PR.